### DISCUSSION TITLE
Does copilot see / consider the other files in opened workspace?

### DISCUSSION THREAD

**Discussion by petya0927** [community/community#13617] [Copilot Conversations] [answered]
It would be nice if copilot could see my other files. So in web developement, as I edit javascript, it could scan my html and give me tips for DOM elements, classes, attributes, etc...

**Comment by davecheney** [comment, marked as answer]
Copilot does this.

**Reply by tmlewin** [reply]
@davecheney i have a question, does it do that in react projects like read imports from other jsx component files  for example?

**Reply by TheGlitch3K** [reply]
Did you ever figure it out, and if so can you explain how to work it?

Thank you,

**Reply by davecheney** [reply]
Just files adjacent to the current file in the ide.
